src/filepattern/java/java.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
Is this file used?

Reply:
Yes, its used to turn on the java bindings on the c++ side from FilePatternBindings.java